test/api/users.coffee

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+User = require("../../website/src/models/user").model
+app = require("../../website/src/server")
+
+describe "Users", ->
+
+  describe "Deleting Users", ->
+
+    before (done) ->
```

Comment:
Well, since the function is set to true, I get the global user that I use for each test.